app/kubernetes-ingress-controller/faq/custom-nginx-conf.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Use a custom nginx.conf file
+
+description: |
+  How do I use a custom nginx.conf file using `ConfigMap` or `Secret`?
+
+breadcrumbs:
+  - /kubernetes-ingress-controller/
+  - index: kubernetes-ingress-controller
+    section: FAQs
+
+content_type: reference
+layout: reference
+
+products:
+  - kic
+
+works_on:
+  - on-prem
+  - konnect
+---
+
+
+You can specify a custom `nginx.conf` file by creating a `ConfigMap` or `Secret` and mounting it inside your container that contains an `nginx_kong.lua` template.
+
+The `deployment.userDefinedVolumes` field in `values.yaml` takes an array of objects that get appended as-is to the existing `spec.template.spec.volumes` array in the {{ site.base_gateway }} deployment resource. The `deployment.userDefinedVolumeMounts` field is appended as-is to the existing `spec.template.spec.containers[].volumeMounts` and `spec.template.spec.initContainers[].volumeMounts` arrays.
```

Comment:
```suggestion
Use the following fields in `values.yaml`:
* `deployment.userDefinedVolumes`: this field takes an array of objects that get appended as-is to the existing `spec.template.spec.volumes` array in the {{ site.base_gateway }} deployment resource. 
* `deployment.userDefinedVolumeMounts`: this field is appended as-is to the existing `spec.template.spec.containers[].volumeMounts` and `spec.template.spec.initContainers[].volumeMounts` arrays.
```
It's really hard to read the paragraph as-is right now with all the code formatting.